src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -0,0 +1,175 @@
+<i:arg name="fieldName" type="String" default="@name"/>
+<i:arg name="value" type="sirius.db.mixing.types.MultiLanguageString"/>
+<i:arg name="labelKey" type="String" default=""/>
+<i:arg name="label" type="String" default="@i18n(labelKey)"/>
+<i:arg name="forceLabel" type="boolean" default="false"/>
+<i:arg name="helpKey" type="String" default=""/>
+<i:arg name="help" type="String" default="@i18n(helpKey)"/>
+<i:arg name="required" type="boolean" default="false"/>
+<i:arg name="readonly" type="boolean" default="false"/>
+<i:arg name="adminOnly" type="boolean" default="false"/>
+<i:arg name="autofocus" type="boolean" default="false"/>
+<i:arg name="type" type="String" default="text"/>
+<i:arg name="id" type="String" default=""/>
+<i:arg name="placeholder" type="String" default=""/>
+<i:arg name="tabIndex" type="String" default=""/>
+<i:arg name="defaultLanguage" type="String" default="@NLS.getCurrentLang()"/>
+<i:arg name="validLanguages" type="Set" default="@value.getValidLanguages()"/>
+
+<i:pragma name="description" value="Renders a multi language text input field within a Wondergem template"/>
+
+<i:local name="functionName" value="@('openMultiLanguageModal' + 'fieldName')"/>
+<i:local name="modalName" value="@('mls-modal-' + fieldName)"/>
+
+<div class="col-xs-@smallSpan col-md-@span form-group @UserContext.get().signalFieldError(fieldName)">
+    <i:if test="isFilled(label) || forceLabel">
+        <label>
+        <span class="@if (required) { input-required } @if (adminOnly) { admin-link }">
+            @label
+        </span>
+        </label>
+    </i:if>
+    <div class="input-group" id="@('insert-point-' + fieldName)">
+        <input @if(isFilled(id)) { id="@id" }
+               type="@type"
+               value="@value.fetchTextOrFallback(defaultLanguage)"
+               class="form-control input-block-level"
+               @if (isFilled(placeholder)) { placeholder="@placeholder" }
+               @if (readonly) { readonly } @if (autofocus) { autofocus }
+               @if (isFilled(tabIndex)) { tabindex="@tabIndex" }/>
+
+        <span class="input-group-addon"><span class="fa fa-globe" onclick="@(functionName+ '()')"></span></span>
+        <div id="@('hidden-inputs-' + fieldName)">
+        </div>
+    </div>
+
+    <i:if test="isFilled(help)">
+        <span class="help-block"><i:raw>@help</i:raw></span>
+    </i:if>
+    <i:if test="UserContext.get().hasError(fieldName)">
+        <span class="error-block help-block">@UserContext.get().getFieldErrorMessage(fieldName)</span>
+    </i:if>
+</div>
+
+<w:modal name="@modalName" title="@label">
+    <w:modalBody>
+        <div class="insert-point">
+
+        </div>
+        <w:modalFooter cancelKey="NLS.cancel">
+            <a class="btn btn-primary ok-btn">@i18n("NLS.ok")</a>
+        </w:modalFooter>
+    </w:modalBody>
+</w:modal>
+<script>
+    let $modal = $("#___modalName");
+    let langObject = JSON.parse(<i:raw>'___value.getAsJSON()'</i:raw>);
```

Comment:
`___value.getAsJSON()` produces NPE, if the MlsEditor is used when creating a new entity

Reply:
Has been fixed in sirius-db